client/extensions/woocommerce/woocommerce-services/lib/initialize-labels-state/index.js

```diff
@@ -50,8 +58,8 @@ export default data => {
 		labels: labelsData || [],
 		paperSize,
 		storeOptions,
-		fulfillOrder: true,
-		emailDetails: true,
+		fulfillOrder: isUndefined( markOrdersComplete ) ? true : markOrdersComplete,
+		emailDetails: isUndefined( emailTrackingInfo ) ? true : emailTrackingInfo,
```

Comment:
So are you suggesting that we remove `fulfillOrder` and `emailDetails` from the initialize state code because the  `QueryLabels` component is used in both the order fulfillment component (in Calypso) and the label view wrapper in the plugin?